scripts/compute_charrnn_vocab.py

```diff
@@ -0,0 +1,43 @@
+from char_vocab_utils import compute_vocab
```

Comment:
Why is this file named charrnn, I would expect that it is model agnostic, that is, computing vocab should general and work for any NN model (charRNN, VAE etc) and SMILE dataset (Zinc, Chembl, Enamine)?

Reply:
the name can be changed. just meant to house all utilities related to the generating of integer-encoded word representations. it is of course agnostic to the specific architecture assuming that this is the expected input of a hypothetical model. will update this to prevent confusion.